src/K9OCRS/Controllers/ClassSectionsController.cs

```diff
@@ -52,8 +52,8 @@ public async Task<IActionResult> GetAll()
                 // Get all sections
                 var sections = await dbOwner.ClassSections.GetAll(conn);
                 // Get all instructors associated to sections
+                //var instructors = (await dbOwner.Users.GetByIDs(conn, instructorId)).ToList();
                 //var instructorIds = sections.Select(s => s.InstructorID);
```

Comment:
This shouldn't have switched places
```suggestion
                //var instructorIds = sections.Select(s => s.InstructorID);
                //var instructors = (await dbOwner.Users.GetByIDs(conn, instructorIds)).ToList();
```